pkg/bootstrapper/kubelet.go

```diff
@@ -183,6 +183,36 @@ func (k *kubeletService) setRecoveryActions() error {
 	return nil
 }
 
+// remove deletes the kubelet service via the Windows service API
+func (k *kubeletService) remove() error {
```

Comment:
Why do you need this method if it is never being used?

Reply:
this method is being used in k.stopAndRemove()

Reply:
I should have been more clear. It is only being used in `k.stopAndRemove()`. Given how little code there is it is fine to just have `k.stopAndRemove()`. But if `k.remove()` was becoming large then yes, you should split it out, which is not the case here.

Reply:
thanks for clarifying, I ll drop this method.